src/vulkan/mod.rs

```diff
@@ -61,6 +62,11 @@ impl Allocation {
         self.chunk_id
     }
 
+    ///Returns the [`vk::MemoryPropertyFlags`](vk::MemoryPropertyFlags) of this allocation.
```

Comment:
No need for a link when the link label/text is the same (intradoc links disregard backticks):

```suggestion
    ///Returns the [`vk::MemoryPropertyFlags`] of this allocation.
```